### PR TITLE
Update Parent Pom

### PR DESCRIPTION
**Pull request by SentryMan**
- imports ebean BOM
- now actually manages the avaje dependencies directly (can use properties to override)
- adds provides maven plugin goals to plugin management

### CODE REVIEW

File: avaje-nima-test/pom.xml [hunk 1]
```diff
@@ -18,23 +18,10 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
```
Comment (rbygrave) [inline avaje-nima-test/pom.xml:21]: Why was this dependency removed?

File: avaje-nima-test/pom.xml [hunk 2]
```diff
@@ -18,23 +18,10 @@
...
     <dependency>
       <groupId>io.avaje</groupId>
       <artifactId>avaje-inject-test</artifactId>
-      <version>${avaje-inject.version}</version>
-    </dependency>
-
-    <dependency>
-      <groupId>io.avaje</groupId>
-      <artifactId>avaje-jsonb</artifactId>
-      <version>${avaje-jsonb.version}</version>
```
Comment (rbygrave) [inline avaje-nima-test/pom.xml:37]: Why was this dependency removed?

File: avaje-nima-test/pom.xml (hunk 1 above)
Comment (SentryMan) [reply avaje-nima-test/pom.xml:21]: It's transitive

Comment (SentryMan) [reply avaje-nima-test/pom.xml:21]: oh wait

File: avaje-nima-test/pom.xml (hunk 2 above)
Comment (SentryMan) [reply avaje-nima-test/pom.xml:37]: it's transitive

File: avaje-nima-test/pom.xml (hunk 1 above)
Comment (SentryMan) [reply avaje-nima-test/pom.xml:21]: fixed

**Comment by SentryMan** [comment]
I wonder if we should instead make a separate Avaje-BOM artifact. In any case, I'm good with this

**Comment by SentryMan** [comment]
@rbygrave please review and deploy an RC when you get time

**Pull request merged**